### ISSUE TITLE
Incomplete Inter-Rater Validation Coverage for VIF Training Data

### ISSUE THREAD

**Issue by DesmondChoy** [DesmondChoy/twinkl#13]
# Problem Summary                                                                               
                                                                                                 
  The current human annotation sample (10 personas, 46 entries) provides **insufficient coverage** 
  across Schwartz value dimensions to validate the reliability of LLM Judge labeling for all
  dimensions.

## Current Coverage Gaps

  | Value Dimension | Personas in Sample | Status |
  |-----------------|-------------------|--------|
  | Self-Direction | 3 | ✓ Adequate |
  | Hedonism | 2 | → Marginal |
  | Conformity | 2 | → Marginal |
  | **Achievement** | **0** | ⚠️ **None** |
  | **Stimulation** | **1** | ⚠️ Insufficient |
  | **Power** | **1** | ⚠️ Insufficient |
  | **Security** | **1** | ⚠️ Insufficient |
  | **Tradition** | **1** | ⚠️ Insufficient |
  | **Benevolence** | **1** | ⚠️ Insufficient |
  | **Universalism** | **1** | ⚠️ Insufficient |

  **Target:** Minimum 3 personas per dimension for reliable κ calculation.

  ---

# Why This Matters for Twinkl

## Academic Claim at Stake

  The VIF training pipeline relies on automated Judge labeling at scale. To justify this
  academically, we must demonstrate that **LLM Judge labeling is at least as reliable as human
  annotation**. This requires robust inter-rater agreement metrics.

  ### Current Validation Results (7 Dimensions Only)

  | Metric | Value | Interpretation |
  |--------|-------|----------------|
  | Human-Human Agreement (Fleiss' κ) | 0.53 | Moderate |
  | Judge-Human Agreement (Avg Cohen's κ) | 0.67 | Substantial |

  **The good news:** Where we *can* measure, the Judge exceeds human-human consistency—its labels
  sit "within the distribution" of human judgments and may capture consensus better than individual
   annotators.

  **The problem:** We can only make this claim for 7 dimensions. The remaining 3 dimensions
  (Achievement, Stimulation, Power) lack the sample size for reliable κ calculation.

 ## Why Core Persona Values Matter

  Coverage must be measured by **personas whose profile includes a dimension as a core value**, not
   just entry-level signal. Personas consistently express their core values across multiple journal
   entries, providing the behavioral patterns annotators need. "Crossover" signal (incidental
  mentions in personas with *other* core values) is:
  - Less consistent
  - Harder to validate against ground truth
  - Unreliable for κ calculation

  ---

# Next Steps

## ✅ Completed
  1. **Synthetic data generation** — Registry contains 102 personas with sufficient coverage (mean
  15.5 personas per value)
  2. **Judge labeling** — All 102 personas labeled

## 🔲 In Progress: Additional Human Annotation Round

  **Requirement:** Annotate **9 additional personas** to reach ≥3 personas per dimension.

  **Optimal Selection (maximizes coverage with minimum effort):**

  | # | Persona | Core Values | Fills Gap For |
  |---|---------|-------------|---------------|
  | 1 | Chen Wei-Lin | Security, Power | Security, Power |
  | 2 | Layla Mansour | Stimulation, Security | Stimulation |
  | 3 | Maya Chen | Achievement, Stimulation | Achievement, Stimulation |
  | 4 | Harold Delacroix | Conformity, Power | Power |
  | 5 | Tariq Al-Mansouri | Benevolence, Universalism | Benevolence, Universalism |
  | 6 | Valentina Reyes | Tradition, Universalism | Tradition, Universalism |
  | 7 | Priya Sharma | Hedonism, Achievement | Achievement |
  | 8 | Tariq Haddad | Benevolence, Achievement | Benevolence |
  | 9 | Park Jiyeon | Tradition, Security | Tradition |

  **Workload:**
  - 9 personas × ~7 entries = **~63 entries** per annotator
  - 3 annotators (des, jl, km) review all entries
  - Personas reordered in registry (`annotation_order` field) — annotators continue from position
  11

## 🔲 Pending: Re-calculate Agreement Metrics

  After annotation completes:

  ```python
  from src.annotation_tool.agreement_metrics import generate_agreement_report
  generate_agreement_report()

  Success Criteria:
  - All 10 dimensions have ≥3 personas with that core value
  - Judge-Human κ ≥ Fleiss' κ (human-human) for all dimensions
  - Update docs/judge_validation_summary.md with complete findings

  ---

## References

  - Analysis Source: src/annotation_tool/agreement_metrics.py
  - Full Report: logs/exports/agreement_report_20260128_133444.md
  - Annotation Tool: src/annotation_tool/

  ---
